frontend/src/components/PageNavBar.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import { Link, useLocation, useRouteMatch } from 'react-router-dom';
+import classNames from 'classnames';
+import css from './PageNavBar.module.scss';
+
+const classes = classNames.bind(css);
+
+interface Header {
+  url: string;
+  text: string;
+}
+
+export const PageNavBar: FC<{
+  headers: Header[];
+}> = ({ headers }) => {
+  const { url } = useRouteMatch();
+  const { pathname } = useLocation();
+  return (
+    <div className={classes(css.navbar)}>
+      {headers.map((header: any) => (
```

Comment:
```suggestion
      {headers.map((header) => (
```
Better keep the parameter typed